standup/auth0/context_processors.py

```diff
@@ -1,32 +1,44 @@
 from urllib.parse import quote_plus
+from uuid import uuid4
 
-from django.conf import settings
+from standup.auth0.settings import app_settings
 
 
 def is_auth0_configured():
     return (
-        settings.AUTH0_CLIENT_ID and
-        settings.AUTH0_CLIENT_SECRET and
-        settings.AUTH0_DOMAIN and
-        settings.AUTH0_CALLBACK_URL
+        app_settings.AUTH0_CLIENT_ID and
+        app_settings.AUTH0_CLIENT_SECRET and
+        app_settings.AUTH0_DOMAIN and
+        app_settings.AUTH0_CALLBACK_URL
     )
 
 
-def build_auth0_url(request):
-    # https://auth0.com/docs/client-auth/server-side-web#calling-the-authorization-url
-    return settings.AUTH0_LOGIN_URL.format(
-        AUTH0_DOMAIN=settings.AUTH0_DOMAIN,
-        AUTH0_CLIENT_ID=quote_plus(settings.AUTH0_CLIENT_ID),
-        AUTH0_CALLBACK_URL=quote_plus(settings.AUTH0_CALLBACK_URL),
-        # FIXME(willkg): This should be a token that ties both ends.
-        STATE='foo',
-    )
+class Auth0URL:
+    """Class that generates an Auth0URL on demand"""
+    def __init__(self, request):
+        self.request = request
+
+    def __str__(self):
```

Comment:
This is nice because one of the side effects of this is that it creates a session. We don't want to create sessions for anyone who hits a page on the site.